apps/src/applab/applab.js

```diff
@@ -960,6 +962,40 @@ Applab.render = function () {
     Applab.reactMountPoint_);
 };
 
+Applab.exportApp = function() {
+  var code = studioApp.editor.getValue();
+  // TODO: find another way to get this element that doesn't rely on global element id.
+  var appElement = document.getElementById('divApplab').cloneNode(true);
+  appElement.style.display = 'block';
+  appElement.classList.remove('notRunning');
+  appElement.classList.remove('withCrosshair');
+  var htmlBody = appElement.outerHTML;
+  var origin = window.location.origin;
+  var html = `
+<html>
+  <head>
+    <title>My App</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="${origin}/assets/js/en_us/common_locale.js"></script>
+    <script src="${origin}/assets/js/en_us/applab_locale.js"></script>
+    <script src="${origin}/assets/js/applab-api.js"></script>
+    <link rel="stylesheet" href="${origin}/assets/css/applab.css">
+  </head>
+  <body>
+    ${htmlBody}
+    <script src="code.js"></script>
+  </body>
+</html>
+`;
```

Comment:
Would you consider replacing this inline HTML block with an ejs template?  It'd be cleaner (and not depend on ES6 multiline strings and interpolation yet).